src/main/java/edu/apigestor/control/controllers/HomeController.java

```diff
@@ -7,11 +7,7 @@
 import edu.apigestor.entity.response.HomeEscolaResponse;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Talvez não faça muito sentido trocar os imports por um wildcard (`*`). Acabamos dificultando entender a origem de algumas anotações e podemos gerar conflito de nomes no futuro (apesar que eu não acredito que deve ocorrer nesse projeto, [boa prática](https://jiga.dev/avoiding-wildcard-imports-in-java-kotlin-with-intellij/) é usar  wildcards nos imports com cautela).

Para manter uma padronização entre os outros códigos, acredito que manter os imports explícitos seja uma melhor alternativa.

Reply:
@Arthursca o quê tu acha? Na minha visão, dificulte a padronização entre as diferentes classes (por exemplo, nos testes unitários eu preciso importar várias classes do `JUnit` que estão no mesmo pacote, se for usar wildcard, pra manter um padrão seria bom usar lá também).